### PR TITLE
create performance charter

### PR DESCRIPTION
**Pull request by chiefcll**

### CODE REVIEW

File: media-performance/problem-statement.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Web Performance on Embedded Devices
+==============================================
+
+Background
+----------
+
+Building a performant and polished UI for a TV application using HTML is extremely difficult. While devices have steadily increased their processing power, the CPU is heavily taxed by Javascript and HTML rendering. The HTML rendering has only gotten worse as users expect a polished interface with gradients, drop shadows, animations and performance of a native app. However, the more complicated the UI the less time there is to use the main thread to do API / business logic work. In order to achieve 60fps, there is 16ms of processing time per frame and the browser will use most of that to do housekeeping work. Even without any business logic, the UI is regularly janky and with business logic and API calls and lazy loading data, the performance of the UI is too poor to ship to customers.
```
Comment (chrisn) [inline media-performance/problem-statement.md:7]: Thanks @chiefcll. The only suggestion I have would be to rephrase the Background slightly to be not so critical, but still highlight the problem.

```suggestion
Building a performant and polished UI for a TV application using HTML is extremely difficult. While devices have steadily increased their processing power, the CPU is heavily taxed by JavaScript and HTML rendering. The performance of HTML rendering is no longer enough to meet users expectations for a polished interface with gradients, drop shadows, animations and performance of a native app. The more complicated the UI the less time there is to use the main thread to do API and business logic work. In order to achieve 60fps, there is 16ms of processing time per frame and the browser will use most of that to do housekeeping work. Even without any business logic, the UI is regularly janky and with business logic and API calls and lazy loading data, the performance of the UI is compromised.
```

**Comment by tidoust** [comment]
Thanks @chiefcll!

A couple of comments:
- The background seems to restrict discussions to WebGL/WebGPU rendering but the envisioned toolkit does not seem specific to that, and use of WebAssembly is only partially related.
- I think the document should be explicit about the deliverables that the task force endeavors to produce. Looking at the list of topics, that would be a toolkit and a document to report on performance considerations, possible approaches to creating performant applications on embedded devices, and possible standardization needs.

I have similar comments on Web Development Experience on Embedded Devices (PR #86):
- The background and problem statement scope discussions to "ways to load a web application on an embedded device" to ease developer experience, but the list of potential topics inludes "Progressive Web Apps" and "Video Playback Layer Support", which are more directly related to converging on a common runtime.
- The document should also be explicit about what the task force will produce. Likely a report that describes current hurdles, needs, and discusses possible standardization approaches to address the needs.

I note that creating two task forces mean twice the overhead (task force moderation, potential calls to progress the work, regular reports to the IG, etc.). I suspect that you might be the initial moderator for both efforts, so I'm wondering whether that's a good idea to divide things into two, especially since there seem to be intersections between the topics that would be explored in both contexts.

In other words, I would like to propose that both proposals be merged into one task force, e.g. "Developing performant web applications for embedded devices". That task force would both cover runtime needs and performance, as well as the developer workflow and debugging environment.

I would be happy to craft a draft charter proposal out of the two proposals you prepared if that sounds like a good plan (adjusting the background section and adding a deliverables section to address my other comments). What about it?

**Comment by chiefcll** [comment]
I'm ok with one or two. Assuming we can have multiple deliverables - creating a better developer experience is more achievable in the short term and would like to get that done.

Please craft up something and I'll review.

**Comment by chrisn** [comment]
One task force sounds good, and we can define it to have multiple deliverables. Thanks @tidoust

**Comment by jpiesing** [comment]
> * I think the document should be explicit about the deliverables that the task force endeavors to produce. Looking at the list of topics, that would be a toolkit and a document to report on performance considerations, possible approaches to creating performant applications on embedded devices, and possible standardization needs.

I agree about being explicit about deliverables. As a device manufacturer, I would like to see some experimental / test programs that device manufacturers / integrators can run on the browsers in their embedded devices to see how well (or badly) the combination of the browser and the hardware performs.

**Comment by chrisn** [comment]
Thanks @jpiesing for your suggestion. We have captured this in the [charter draft](https://github.com/w3c/media-and-entertainment/blob/master/app-development/charter.md).

**Comment by chrisn** [comment]
I'll close this as superseded by Francois' draft. We welcome additional review feedback, feel free to raise new issues.
